src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/ModuleScaffold.kt

```diff
@@ -105,6 +104,7 @@ object ModuleScaffold : Module("Scaffold", Category.WORLD) {
 
     // Rotation
     private val rotationsConfigurable = tree(RotationsConfigurable())
+    private val jumpFactor by float("jumpFactor", 0.7f, 0f..2f);
```

Comment:
"JumpFactor"

Reply:
Already mentioned that on Guilded + description of what it actually does is totally different from term "JumpFactor".

Reply:
As I said on Guilded, it was only a very quick mock op to show that it worked, but i have changed it to something clearer now